src/main/java/kafdrop/config/KafkaConfiguration.java

```diff
@@ -48,12 +52,16 @@ public void applyCommon(Properties properties) {
     }
 
     LOG.info("Checking properties file {}", propertiesFile);
-    final var propertiesFile = new File(this.propertiesFile);
-    if (propertiesFile.isFile()) {
-      LOG.info("Loading properties from {}", this.propertiesFile);
+    Optional<AbstractResource> readablePropertyFile = StringUtils.isBlank(propertiesFile) ? Optional.empty() :
```

Comment:
```suggestion
    Optional<AbstractResource> propertiesResource = StringUtils.isBlank(propertiesFile) ? Optional.empty() :
```

Reply:
@Bert-R Thanks for the feedback. I've changed the variable name.